src/lib/relicScorer.ts

```diff
@@ -340,33 +340,21 @@ export class RelicScorer {
     }
   }
 
-  static scoreRelic(relic, id, mainStatScoring = 'ideal', withMeta = false) {
-    return new RelicScorer().scoreRelic(relic, id, mainStatScoring, withMeta)
+  static scoreRelic(relic, id, withMeta = false) {
+    return new RelicScorer().scoreRelic(relic, id, withMeta)
   }
 
-  scoreRelic(relic: Relic, id: CharacterId, mainStatScoring: string = 'ideal', withMeta: boolean = false) {
```

Comment:
I think there's a usage of this in relicScorer.test.ts - it passes `undefined` to opt into the default
(run with `npm run vitest` iirc)

Reply:
Just fixed that - thanks!